lib/utils/process.js

```diff
@@ -11,6 +11,8 @@ exports.exec = function(cmd, options, callback) {
 
 	var spawnOptions = _(options).clone();
 
+	spawnOptions.env = _({}).defaults(options.env, process.env);
```

Comment:
Not sure if this is a breaking change or not

Reply:
Seems to be a breaking change for me

Reply:
Yes, looks like breaking change, so it will be 5.0.0